assignments/assignment05.rb

```diff
@@ -6,6 +6,42 @@
 #  Problem 1 -  implement prop_reader, write MiniTest unit tests
 
 # expected results:
+
+require 'minitest/autorun'
+
+class Class
+	#prop_reader :flavor
+
+	def prop_reader(*flavors)
```

Comment:
kinda weird that you are calling your argument `flavors` instead of `props` or `properties`. But the code looks correct.